app/proguard-files/proguard-rules.pro

```diff
@@ -0,0 +1,56 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in /home/valeria/android-sdk-linux/tools/proguard/proguard-android.txt
```

Comment:
This is auto generated?  Not sure this has a huge impact on anything but I feel its better to avoid having such a path reference if possible

Reply:
yes probably generated automatically, I removed it